ehcache-xml/src/common/schema/ehcache-core.xsd

```diff
@@ -538,6 +540,9 @@
       </xs:appinfo>
       <xs:appinfo>
         <annox:annotate>@java.lang.SuppressWarnings({"unchecked", "serial"})</annox:annotate>
+        <xs:documentation>
```

Comment:
For this element we should be able to lift the comment from the other `heap` element.
```

DEPRECATED: The use of memory units (such as 'B', 'kB' or 'MB') for heap resources is deprecated and will be removed in a future version.
```

Reply:
Accommodated - Placed same statement